relationships/synthesis/INFRA-HOST-to-MLOPS-MACHINE_LEARNING_MODEL.yml

```diff
@@ -0,0 +1,65 @@
+relationships:
+  - name: mlEndpointHostsModelFromEndpointName
```

Comment:
for my own tracking purposes, originally this was named `mlEndpointHostsModel`

Reply:
My idea is to use this method not to have numbers as we defined in the past: mlEndpointHostsModel1 and 2.